tex/gregoriotex-main.tex

```diff
@@ -678,7 +575,10 @@
 %% has to be called before drawing the key, after drawing the initial
 \def\gre@beginnotes{%
   \gre@trace{gre@beginnotes}%
-  \gre@drawfirstlines %
+  % Draw the staff lines the same as the other staves; post_linebreak
+  % will fix them. The \hbox is there so that post_linebreak can find
```

Comment:
This is a hack. It would be better either to mark the initial with an attribute, or to draw all the lines using \localleftbox (I couldn't figure out how).